docs/03.API-EXAMPLE.md

```diff
@@ -283,6 +284,12 @@ main (int argc, char * argv[])
       len++;
     }
 
+    /* If the command starts with 'quit', break the loop */
+    if (!strncmp (cmd, "quit", strlen ("quit")))
```

Comment:
But something like quitten also quits. Perhaps we could check "quit" or "quit\n" and be happy.